shared/people/container.js

```diff
@@ -0,0 +1,131 @@
+// @flow
+import People from './'
+import * as PeopleGen from '../actions/people-gen'
+import * as Types from '../constants/types/people'
+import * as Tabs from '../constants/tabs'
+import * as SettingsTabs from '../constants/settings'
+import {createGetMyProfile} from '../actions/tracker-gen'
+import {connect} from 'react-redux'
+import {type TypedState} from '../util/container'
+import {createSearchSuggestions} from '../actions/search-gen'
+import {navigateAppend, switchTo, navigateTo} from '../actions/route-tree'
+import {createShowUserProfile} from '../actions/profile-gen'
+import openURL from '../util/open-url'
+import {isMobile} from '../constants/platform'
+import {getPeopleDataWaitingKey} from '../constants/people'
+// import flags from '../util/feature-flags'
+
+const INSTALL_LINK_URL = 'https://keybase.io/download'
```

Comment:
here and above we don't really use this form that much, likely better to just make it camelcase